pom.xml

```diff
@@ -352,6 +366,7 @@
                 <configuration>
                     <excludes>
                         <exclude>integration/*.java</exclude>
+                        <exclude>benchmarks/**/*</exclude>
```

Comment:
Since my test doesn't follow maven name conventions(i.e doesn't end with Test) it will be excluded without this line.
But I added this anyhows :-P.